src/solc_0.7/common/BaseWithStorage/ERC721BaseToken.sol

```diff
@@ -238,6 +238,9 @@ contract ERC721BaseToken is IERC721, WithSuperOperators, WithMetaTransaction {
         return id == 0x01ffc9a7 || id == 0x80ac58cd;
     }
 
+    /// @dev By overriding this function in an implementation which inherits this contract, you can enable versioned tokenIds without the extra overhead of writing to a new storage slot in _owners each time a version is incremented. See GameToken._storageId() for an example, where the storageId is the tokenId minus the version number.
```

Comment:
I would add a comment that by doing that you without extra precaution, you ll have ownerOf return an owner for non existing token and that test should be written to ensure this is not the case

Reply:
done.